web/swagger.json

```diff
@@ -3116,12 +3116,17 @@
         },
         "command": {
             "required": [
-                "commandId"
+                "commandId",
+                "uuid"
```

Comment:
@LucWollants Niet alle commands geven nu een uuid terug. Enkel bij het aanmaken van een nieuwe _Offer_ zit er een  _eventId_ of _placeId_ bij in de response.